pkg/cosign/cosign.go

```diff
@@ -279,6 +279,20 @@ func fetchAttestations(opts Options) (*Response, error) {
 		return nil, err
 	}
 
+	payload, err := extractPayload(signatures)
+	if err != nil {
+		return nil, err
+	}
+
```

Comment:
Do we need `matchSubjectAndIssuer` check as Jim pointed out [here](https://github.com/kyverno/kyverno/issues/4668#issuecomment-1264998258)?

```
if err := matchSubjectAndIssuer(signatures, opts.Subject, opts.Issuer); err != nil {
		return nil, err
	}
```

Reply:
@realshuting [matchSubjectAndIssuer](https://github.com/kyverno/kyverno/blob/v1.7.4/pkg/cosign/cosign.go#L420) in v1.7.4 method has been modified to [matchCertificate](https://github.com/kyverno/kyverno/blob/main/pkg/cosign/cosign.go#L434) method in the `main` branch. Although this also checks the subject and issuer match.